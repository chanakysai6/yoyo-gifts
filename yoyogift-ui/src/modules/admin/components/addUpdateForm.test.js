import React from "react";
import  { shallow  , configure
 } from "enzyme";
 import { Provider } from "react-redux";
import configureMockStore from "redux-mock-store";
 import Adapter from 'enzyme-adapter-react-16';
import AddUpdateForm from "./addUpdateForm";

configure({adapter: new Adapter()})
const mockStore = configureMockStore();
const store = mockStore({});

// const formState = {
//   cardNameValue: "",
//   cardNameError: false,
//   cardNameHelperText: "",
//   cardPointsValue: "",
//   cardPointsError: false,
//   cardPointsHelperText: "",
//   cardCategoryValue: "",
//   cardCategoryError: false,
//   cardCategoryHelperText: "",
//   cardRetailerValue: "",
//   cardRetailerError: false,
//   cardRetailerHelperText: "",
//   cardExpiryDateValue: "",
//   cardExpiryDateError: false,
//   cardExpiryDateHelperText: "",
//   cardCountValue: "",
//   cardCountError: false,
//   cardCountHelperText: "",
//   cardImageValue: "",
//   cardImageError: false,
//   cardImageHelperText: "",
//   cardVendorValue: "",
//   cardVendorError: false,
//   cardVendorHelperText: "",
//   cardShortDescValue: "",
//   cardShortDescError: false,
//   cardShortDescHelperText: "",
//   cardLongDescValue: "",
//   cardLongDescError: false,
//   cardLongDescHelperText: ""
// };
describe("Testing AddUpdate form", () => {
  it("Snapshot testing", () => {
    const wrapper = shallow(
      <Provider store={store}>
          <AddUpdateForm 
        giftCards={[{ id: 1, cardName: "test" }, {}, {}]}
        match={{ params: { id: true } }}
      />
      </Provider>
    );
    expect(wrapper).toMatchSnapshot();
  });
});