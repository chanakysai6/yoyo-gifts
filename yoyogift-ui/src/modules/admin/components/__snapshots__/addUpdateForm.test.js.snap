// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Testing AddUpdate form Snapshot testing 1`] = `ShallowWrapper {}`;
